README.md

```diff
@@ -133,42 +189,63 @@ Modified: GET /consommations
   Response modified: 200
     [Breaking] Body attribute modified: energie
 ```
+> You can create as many diffs as you like without being authenticated. This is a **free and unlimited service** provided you have a fair use of the service.
+{: .info}
 
-Or from two URLs:
+_Note: You can also test this feature in our dedicated web application at <https://api-diff.io/>._
+
+#### Authenticated diffs related to your Bump.sh documentation
+
+From an existing Bump.sh documentation, the `diff` command will retrieve a comparison changelog between your latest published documentation and the given file or URL:
 
 ```sh-session
-$ bump diff https://demo.bump.sh/doc/trips-books/changes/bfec0a43-b870-44da-9e07-60c8955e15d5.json https://demo.bump.sh/doc/trips-books.json
-* Comparing the two given definition files... done
-Modified: POST /books
-  Response modified: 200
-    [Breaking] Body attribute removed: cent
+bump diff path/to/your/file.yml --doc my-documentation --token $DOC_TOKEN
 ```
 
-_Note: You can also test this feature in our dedicated web application at <https://api-diff.io/>._
+If you want to compare two unpublished versions of your API document, the `diff` command can retrieve a comparison changelog between two given file or URL, “as simple as `git diff`”:
+
+```sh-session
+bump diff path/to/your/file.yml path/to/your/next-file.yml --doc my-documentation --token $DOC_TOKEN
+```
 
-#### Authenticated diffs attached to your Bump documentation
+Please check `bump diff --help` for full usage details.
 
-From a Bump documentation, the `diff` command will retrieve a comparison changelog between your existing documentation and the given file or URL:
+### `bump preview [FILE]`
 
-```sh-session
-$ bump diff path/to/your/file.yml --doc DOC_ID_OR_SLUG --token DOC_TOKEN
-* Comparing the given definition file with the currently deployed one... done
 
-Updated: POST /validations
-  Body attribute modified: documentation
+When writing a documentation, you might want to preview how it renders on Bump.sh. This is exactly the goal of the `preview` command: it will create a temporary documentation with a unique URL, which will be available for a short period of time.
```

Comment:
```suggestion
When writing a documentation, you might want to preview how it renders on Bump.sh. This is precisely the goal of the `preview` command: it will create a temporary documentation with a unique URL, which will be available for a short period.
```